pkg/printers/internalversion/printers.go

```diff
@@ -60,7 +59,20 @@ import (
 	"k8s.io/kubernetes/pkg/util/node"
 )
 
-const loadBalancerWidth = 16
+const (
+	loadBalancerWidth = 16
+
+	// LabelNodeRoleMaster specifies that a node is a master
+	// It's copied over to here until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
+	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
@philips @jbeda  ^

@thockin has been discussed in various forums, can see if I can dig something up; but basically it gives us the possibility to eventually easily support multiple well-known roles _for the same node_.
i.e. can be both master (`"node-role.kubernetes.io/master"`) and whatever else we might come up with.